error while almost simultaneous rospy get/set params

Testing simple.html with current release causes this error. It seems like changes in rospy causes error.

```
Traceback (most recent call last):
  File "/opt/ros/hydro/lib/python2.7/dist-packages/rospy/impl/tcpros_service.py", line 618, in _handle_request
    response = convert_return_to_response(self.handler(request), self.response_class)
  File "/home/jihoonl/research/ros/isolate/src/rosbridge_suite/rosapi/scripts/rosapi", line 135, in set_param
    rosapi.params.set_param(request.name, request.value)
  File "/home/jihoonl/research/ros/isolate/src/rosbridge_suite/rosapi/src/rosapi/params.py", line 53, in set_param
    rospy.set_param(name, d)
  File "/opt/ros/hydro/lib/python2.7/dist-packages/rospy/client.py", line 494, in set_param
    _param_server[param_name] = param_value #MasterProxy does all the magic for us
  File "/opt/ros/hydro/lib/python2.7/dist-packages/rospy/msproxy.py", line 140, in __setitem__
    self.target.setParam(rospy.names.get_caller_id(), rospy.names.resolve_name(key), val)
  File "/usr/lib/python2.7/xmlrpclib.py", line 1224, in __call__
    return self.__send(self.__name, args)
  File "/usr/lib/python2.7/xmlrpclib.py", line 1578, in __request
    verbose=self.__verbose
  File "/usr/lib/python2.7/xmlrpclib.py", line 1264, in request
    return self.single_request(host, handler, request_body, verbose)
  File "/usr/lib/python2.7/xmlrpclib.py", line 1289, in single_request
    self.send_request(h, handler, request_body)
  File "/usr/lib/python2.7/xmlrpclib.py", line 1391, in send_request
    connection.putrequest("POST", handler, skip_accept_encoding=True)
  File "/usr/lib/python2.7/httplib.py", line 856, in putrequest
    raise CannotSendRequest()
CannotSendRequest
[ERROR] [WallTime: 1398643081.784109] Error processing request: 
None
[ERROR] [WallTime: 1398643081.784417] [Client 11] [id: call_service:/rosapi/set_param:2] call_service ServiceException: service [/rosapi/set_param] responded with an error: error processing request: 
```

--- a/rosbridge_library/src/rosbridge_library/internal/ros_loader.py
+++ b/rosbridge_library/src/rosbridge_library/internal/ros_loader.py
@@ -49,6 +49,7 @@ _loaded_msgs = {}
 _loaded_srvs = {}
 _msgs_lock = Lock()
 _srvs_lock = Lock()
+_manifest_lock = Lock()
 
 
 class InvalidTypeStringException(Exception):
@@ -179,11 +180,11 @@ def _load_class(modname, subname, classname):
     global loaded_modules
 
     try:
-        # roslib maintains a cache of loaded manifests, so no need to duplicate
-        roslib.launcher.load_manifest(modname)
+        with _manifest_lock:
+            # roslib maintains a cache of loaded manifests, so no need to duplicate
+            roslib.launcher.load_manifest(modname)
     except Exception as exc:
         raise InvalidPackageException(modname, exc)
-
 
     try:
         pypkg = __import__('%s.%s' % (modname, subname))

